### ISSUE TITLE
Escaping Characters with \  confusing

### ISSUE THREAD

**Issue by StefanPp** [mike-ward/Markdown-Edit#261]
Hi guys,
best Editor ever, but one thing seems to be strange. Why do you escape some special chars?`These backslahes are _shown_ on other editors. I work parallel on NextCloud Notes-Addon. E.g. checkboxes look like
`  - \[ \] my note\!`
wich is not really nice.
Any chance do disable that?

**Comment by mike-ward** [comment]
Is this what you're referring to? 

https://spec.commonmark.org/0.27/#backslash-escapes

**Comment by StefanPp** [comment]
You are right. This maybe according to CommonMark, but as far as I understood, MD files should be clearly readable with any text editor as well.
I testet a few editors and even MarkUp editors show the backslashes.

**Comment by StefanPp** [comment]
Additionally I found out, that the automatic Line Break (WordWrap) breaks puts only one word in next line if an empy line follows:

```
word word word word word word word word word word word word word word
word word
word

```
it's impossible to leave the last 3 words in the 2nd line.
